.travis.yml

```diff
@@ -13,22 +13,27 @@ python:
   - "2.7"
 
 env:
-  - VERSION="7.0" ODOO_REPO="odoo/odoo"
-  - VERSION="7.0" ODOO_REPO="OCA/OCB"
+  global:
+  - VERSION="7.0" ODOO_REPO="odoo/odoo" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
+  - TRANSIFEX_USER='transbot@odoo-community.org'
+  - secure: vSumRFP7qtgd8qERNnGwxxlb8hxPUyks3tUJ3EKOUE88PKEb/0ofvfjtyNa8ZHN+OMl7exod3Nuh1ejYw6uMfw8WQYjjMyPJL7sf8OK36CMU0drlj1m4/ohWn4BRyDeHL2GAymDG/TEHMNgY2s1Y2ASC9vhXOyzaj95VQhZH8ak=
```

Comment:
This seems wrong, it refers to stock-logistics-transport instead not to stock-logistics-barcode